python-backend/app/__init__.py

```diff
@@ -26,9 +27,14 @@ def create_app(test_config=None):
         # load the test config if passed in
         app.config.from_object(test_config)
 
+    configure_uploads(app, documents)
+    #convert to bytes from MB
+    patch_request_class(app, (Config.FILE_BYTE_LIMIT*1024*1024))
```

Comment:
You can also achieve the same behavior by setting the `MAX_CONTENT_LENGTH` app config variable for this setting. patch_request_class which is just there for backwards compatibility if we were using an older version of flask.
See the Configuration Tab
https://flask-uploads.readthedocs.io/en/latest/

Reply:
Changed this to use flasks built in MAX_CONTENT_LENGTH